BrainPortal/public/javascripts/cbrain.js

```diff
@@ -1016,6 +1016,27 @@
     }
   });
 
+    // Add select/deselect all toggle to optgroups in chosen
```

Comment:
This seems to work but I suspect this code was added from some source on the internet? Then please add a comment with the source and make sure we're allowed to use that code.

Reply:
Yep see my comment 4 days ago:

> I found a pretty nice solution:
> 
> * http://jsfiddle.net/jphase/A6LBv/
> 
> The code just need to be added in the `cbrain.js` a good place to do that is around l1010

Reply:
Yes, when I wrote 'I suspect' I really meant to say I'd seen you had provided a URL before, though I did no follow the link myself. Anyway, if Jeff Hays is the author, ask him if we can use the snippet, and credit him in the code.

Reply:
Hey Team,

This is Jeff Hays, the author of the aforementioned snippet. You are more than welcome to use my code and I’m happy to see it being put to use! Let me know if there’s anything else I can help contribute to.

Reply:
@jphase Hi Jeff! We'll include one comment line with your name and GitHub handle directly in the code. Thanks!

@natacha-beck since Candice is away on vacation can you do it please? Thanks.